Hangman/Hangman/main.swift

```diff
@@ -2,11 +2,167 @@
+let hangManDrawing = ["""
+-----|
+|
+|
+|
+|
+|
+--------
+""",
+                      """
+-----|
+|    O
+|
+|
+|
+|
+--------
+""",
+                      """
+-----|
+|    O
+|    |
+|
+|
+|
+--------
+""",
+                      """
+-----|
+|    O
+|    |\\
+|
+|
+|
+---------
+""",
+                      """
+-----|
+|    O
+|   /|\\
+|
+|
+|
+---------
+""",
+                      """
+-----|
+|    O
+|   /|\\
+|     \\
+|
+|
+---------
+""",
+                      """
+-----|
+|    O
+|   /|\\
+|   / \\
+|
+|
+---------
+"""
+]
 
-print("Hello, World!")
+//var gameState = true
+//var gameRestart = false
+var repeatOnlyForAttempts = true
 
+
+func Hangman() {
+
+    var numOfLosses = 0
+    var numOfAttemptsRemaining = 6
+    let randomAiWord = allTheWords.randomElement()
+    var validAiWordAsString = ""
+    
+    if let validRandomAiWord = randomAiWord {
```

Comment:
You did a wonderful job naming your variables, I was a little confused by random"Ai"Word but otherwise, absolutely great! The idea behind name your variables is that everyone who you're working with should be able to read your code and be able to understand it :)

Reply:
Its fine, by "Ai", I meant what element the "pseudorandom algorithm" or "Ai" picked. But, I agree that randomWord is better to understand